### ISSUE TITLE
HvDatePicker (Control the OnClick of the HvDatePicker) - https://lumada-design.github.io/uikit/master/?path=/docs/components-date-picker--main.

### ISSUE THREAD

**Issue by sahillihas2021** [lumada-design/hv-uikit-react#3686]
### Which version of NEXT UI Kit are you opening this bug for?

v5.x

### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

As per the current scenario - I have a button labelled as Custom. 

On click of the 'Custom' button, It opens the HvDatePicker (Datepicker in range mode allowing the selection of more than one value.)

![Current](https://github.com/lumada-design/hv-uikit-react/assets/93397536/a456d42e-6038-45ff-b35f-e8deae13bade)

### Expected behavior 🤔

As per the current scenario - I have a button labelled as Custom. 

On click of the 'Custom' button, It opens the HvDatePicker (Datepicker in range mode allowing the selection of more than one value.) along with the inside view of the date range selection view.

![Required](https://github.com/lumada-design/hv-uikit-react/assets/93397536/18fa4fc1-9222-4340-9f4a-f17d96a030e2)

### Steps to reproduce 🕹

Enter sandbox link: https://stackblitz.com/edit/github-xysspw?file=src%2FApp.tsx

And/or the steps to reproduce:

1. Just run the code present in the App.tsx file.
2. In the output you will have a custom button.
3. Upon click of the button alternatively it will open/close the HvDatePicker.

### Context 🔦

I wanted to open the inside view of the HvDatePicker (Range Selection) as well upon click on the custom button.

Basically - I want to programmatically onclick the HvDatePicker to show the date range selection view as well upon click of the custom button [From outside of it - it might be a button, dropdown].

You can let me know the steps/approach to get the desired result.
Thankyou !

### Your Environment 🌎

Device: Laptop
OS: Windows 10
Browser: chrome
Version 116.0.5845.188 (Official Build) (64-bit)
Other details: screen resolution 1920x1080

**Comment by sahillihas2021** [comment]
Hi Team
Any update on this ?

**Comment by zettca** [comment]
Hello @sahillihas2021 👋 

Apologies for the delay. `HvDatePicker` currently doesn't support control of the expanded/collapsed state, unlike similar components. We'll be adding it

As a workaround for now, you can trigger the toggle with an `click()` on the header element

**Comment by zettca** [comment]
Hey @sahillihas2021 

We've added the `expanded`/`defaultExpanded` props allow this control. Expect the release tomorrow morning 🚀
